metric_names/command.go

```diff
@@ -0,0 +1,53 @@
+package metric_names
+
+import (
+	"os"
+
+	"github.com/mackerelio/mkr/format"
+	"github.com/mackerelio/mkr/jq"
+	"github.com/mackerelio/mkr/logger"
+	"github.com/mackerelio/mkr/mackerelclient"
+	"github.com/urfave/cli"
+)
+
+var Command = cli.Command{
+	Name:      "metric-names",
+	Usage:     "Fetch metric names",
+	ArgsUsage: "[--host | -H <hostId>] [--service | -s <service>] [--jq <formula>]",
+	Description: `
+    Fetch metric names of 'host metric' or 'service metric'.
+    Requests "/api/v0/hosts/<hostId>/metric-names" or "/api/v0/services/<serviceName>/metric-names".
+    See https://mackerel.io/ja/api-docs/entry/hosts#metric-names, https://mackerel.io/ja/api-docs/entry/services#metric-names
```

Comment:
Please link to english documentation.

```suggestion
    See https://mackerel.io/api-docs/entry/hosts#metric-names, https://mackerel.io/api-docs/entry/services#metric-names
```